api_facade.py

```diff
@@ -0,0 +1,290 @@
+courses = {
+    'swen-101':
+        {
+            "courseName": "Freshman Seminar",
+            "courseId": "swen-101",
+            "numGrades": 0,
+            "numStudents": 60,
+            "instructor": "0001",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
```

Comment:
Just some small things here we may or may not want to address..

- All fields with relations are probably going to be to the foreign key / id of that entity. So we should probably follow that convention here so there's minimal rework for the differences in what the API will be giving us. 
- Also, some of these fields can be calculated based on the data and I don't think the API will be returning that directly, such as `numStudents` and `numGrades`. But we can get `numStudents` from the list of `students` and we can get `numGrades` by requesting all grades for a given `course.id`.